rotkehlchen/chain/ethereum/modules/curve/decoder.py

```diff
@@ -502,13 +505,32 @@ def _decode_curve_events(self, context: DecoderContext) -> DecodingOutput:
         return DEFAULT_DECODING_OUTPUT
 
     def _decode_curve_gauge_events(self, context: DecoderContext) -> DecodingOutput:
-        if context.tx_log.topics[0] not in (GAUGE_DEPOSIT, GAUGE_WITHDRAW):
+        if context.tx_log.topics[0] not in (GAUGE_DEPOSIT, GAUGE_WITHDRAW, GAUGE_VOTE):
             return DEFAULT_DECODING_OUTPUT
 
-        provider = hex_or_bytes_to_address(context.tx_log.topics[1])
+        found_event_modifying_balances = False
         gauge_address = context.tx_log.address
+        if context.tx_log.topics[0] == GAUGE_VOTE:
+            gauge_address = hex_or_bytes_to_address(context.tx_log.data[64:96])
+            # create the vote event (fee event is already created)
```

Comment:
This comment makes no sense. It's obvious you are creating the vote event. I don't know what you mean by fee event?

```suggestion
```